contracts/lease/src/api/open/mod.rs

```diff
@@ -153,6 +153,16 @@ impl PositionSpec {
         obj
     }
 
+    pub fn validate_open_amount(self, amount: LpnCoin) -> ContractResult<()> {
```

Comment:
1. Why `self`?
2. The fn name implies validation of the position amount and that is not true as per the implementation